src/main/groovy/org/pih/warehouse/inventory/CycleCountUpdateItemCommand.groovy

```diff
@@ -0,0 +1,25 @@
+package org.pih.warehouse.inventory
+
+import grails.validation.Validateable
+import org.springframework.web.context.request.RequestContextHolder
+
+class CycleCountUpdateItemCommand implements Validateable {
```

Comment:
Same point I made in Evan's PR. We should go with action or resource command naming convention. 

UpdateCycleCountItemCommand would be my preference. CycleCountItemCommand would be the other approach, which would make sense if we didn't need to distinguish between creating and updating cycle count item.

Reply:
I personally like the CycleCount prefix since it gives us a clear grouping of commands by feature. Feature + action. I think this is something we should standardize through discovery since we're definitely not consistent about it.